### PR TITLE
Formatting (plot_ci function)

### PR DESCRIPTION
**Pull request by zackt113**
Just fixed the formatting issue.

### CODE REVIEW

**Review by voremargot — COMMENTED** [review]
Looks good to me!

File: R/plot_ci.R [hunk 1]
```diff
@@ -1,44 +1,38 @@
...
-plot_ci <- function(sample, rep, bin_size = 30, 
-                              n = "auto",ci_level = 0.95, 
-                              ci_random_seed = NULL, 
-                              title = "Bootstrap Sampling Plot", 
-                              x_axis = "Bootstrap Sample Mean",
-                              y_axis = "Count", 
-                              save_result_to = ".") {
+plot_ci <- function(sample, rep, bin_size = 30, n = "auto", ci_level = 0.95, 
+                    ci_random_seed = NULL, title = "Bootstrap Sampling Plot", 
+                    x_axis = "Bootstrap Sample Mean",y_axis = "Count",
```
Comment (gauthampughazhendhi) [inline R/plot_ci.R]:
```suggestion
                    x_axis = "Bootstrap Sample Mean", y_axis = "Count", 
```

File: R/plot_ci.R [hunk 2]
```diff
@@ -70,53 +62,42 @@ plot_ci <- function(sample, rep, bin_size = 30,
...
-                      label = (toString(round(unname(sample_stat_dict$upper), 2))), 
-                      size = 3, 
-                      color = "purple",
-                      fill=" lightgrey") +
+    ggplot2::annotate(geom="label", x = sample_stat_dict$sample_mean, 
+                      y = 0.9 * y_max, label = paste(str_1, "+/-", str_2), 
+                      size = 3, color = "red", fill=" lightgrey") + 
+    ggplot2::annotate(geom="label",  x = unname(sample_stat_dict$lower), 
+                      y = 0.9 * y_max, label = lower_label, size = 3, 
+                      color = "purple", fill=" lightgrey") +
```
Comment (gauthampughazhendhi) [inline R/plot_ci.R]:
```suggestion
                      color = "purple", fill = " lightgrey") +
```

File: R/plot_ci.R [hunk 3]
```diff
@@ -70,53 +62,42 @@ plot_ci <- function(sample, rep, bin_size = 30,
...
-    ggplot2::annotate(geom="label", 
-                      x = unname(sample_stat_dict$upper), 
-                      y = 0.9 * ggplot2::ggplot_build(bootstrap_dist_ci)$layout$panel_params[[1]]$y.range[2], 
-                      label = (toString(round(unname(sample_stat_dict$upper), 2))), 
-                      size = 3, 
-                      color = "purple",
-                      fill=" lightgrey") +
+    ggplot2::annotate(geom="label", x = sample_stat_dict$sample_mean, 
+                      y = 0.9 * y_max, label = paste(str_1, "+/-", str_2), 
+                      size = 3, color = "red", fill=" lightgrey") +
```
Comment (gauthampughazhendhi) [inline R/plot_ci.R]:
```suggestion
                      size = 3, color = "red", fill = " lightgrey") + 
```

File: R/plot_ci.R [hunk 4]
```diff
@@ -70,53 +62,42 @@ plot_ci <- function(sample, rep, bin_size = 30,
...
-                      fill=" lightgrey") +
+    ggplot2::annotate(geom="label", x = sample_stat_dict$sample_mean, 
+                      y = 0.9 * y_max, label = paste(str_1, "+/-", str_2), 
+                      size = 3, color = "red", fill=" lightgrey") + 
+    ggplot2::annotate(geom="label",  x = unname(sample_stat_dict$lower), 
+                      y = 0.9 * y_max, label = lower_label, size = 3, 
+                      color = "purple", fill=" lightgrey") +
+    ggplot2::annotate(geom="label", x = unname(sample_stat_dict$upper), 
+                      y = 0.9 * y_max, label = upper_label, size = 3, 
+                      color = "purple", fill=" lightgrey") +
```
Comment (gauthampughazhendhi) [inline R/plot_ci.R]:
```suggestion
                      color = "purple", fill = " lightgrey") +
```

**Review by gauthampughazhendhi — CHANGES_REQUESTED** [review]
Added some suggestions

**Comment by zackt113** [comment]
Accepted! Thanks!

**Pull request merged**